validations/src/tests/mod.rs

```diff
@@ -6486,6 +6607,222 @@ fn tally_unserializable_value() {
     x.unwrap();
 }
 
+#[test]
+fn tally_unhandled_intercept_with_message() {
+    // Reveals with value RadonErrors::UnhandledIntercept are accepted, but the message field must
+    // be removed from the tally transaction.
+
+    // Reveal value: 39([255, "Hello!"])
+    let reveal_value = vec![
+        216, 39, 130, 24, 255, 0x66, b'H', b'e', b'l', b'l', b'o', b'!',
+    ];
+    let dr_output = example_data_request_output(2, 200, 20);
+    let (dr_pool, dr_pointer, _rewarded, slashed, error_witnesses, _dr_pkh, change, reward) =
+        dr_pool_with_dr_in_tally_all_errors(dr_output, 2, 2, reveal_value.clone());
+    assert_eq!(reward, ONE_WIT + 200);
+
+    // Tally value with message: 39([255, "Hello!"])
+    let tally_value_with_message = reveal_value;
+    // Tally value no message: 39([255])
+    let tally_value_no_message = vec![216, 39, 129, 24, 255];
+    let vt0 = ValueTransferOutput {
+        time_lock: 0,
+        pkh: error_witnesses[0],
+        value: reward,
+    };
+    let vt1 = ValueTransferOutput {
+        time_lock: 0,
+        pkh: error_witnesses[1],
+        value: reward,
+    };
+    assert_eq!(change, 0);
+    let tally_transaction_with_message = TallyTransaction::new(
+        dr_pointer,
+        tally_value_with_message.clone(),
+        vec![vt0.clone(), vt1.clone()],
+        slashed.clone(),
+        error_witnesses.clone(),
+    );
+    let tally_transaction_no_message = TallyTransaction::new(
+        dr_pointer,
+        tally_value_no_message.clone(),
+        vec![vt0, vt1],
+        slashed,
+        error_witnesses,
+    );
+
+    let mut active_wips = all_wips_active();
```

Comment:
in RNG PR (#2056) I use all_wips_active to have activated also wips in voting process, and in this case I use current_active_wips()